From d0f34ce92b65dd363b619528215927a3d7878163 Mon Sep 17 00:00:00 2001
From: chen xu <fionaxu@google.com>
Date: Tue, 9 Oct 2018 18:53:00 -0700
Subject: [PATCH 2/3] set config_enabled_lte to true by default

AOSP by default set this flag off, thus on AOSP/GSI build, device can't
set preferred NW to 4G. We should trun this on by default. If OEMs dont
want 4G options, they can explicitly disable it in resource overlay

Bug: 117133176
Test: telephony unit test
Test: boot a GSI and checks LTE appears in "Preferred network type".
Change-Id: I109631e422b712fa16e8ad40f467439adaaafc08
(cherry picked from commit 9eb662f6010b48945141e0c3b0869d276e5c7751)
---
 res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values/config.xml b/res/values/config.xml
index ba94790c3..d3f24f1e5 100644
--- a/res/values/config.xml
+++ b/res/values/config.xml
@@ -137,7 +137,7 @@
     <!-- DEPRECATED: Use CarrierConfigManager#KEY_PREFER_2G_BOOL -->
     <bool name="config_prefer_2g" translatable="false">true</bool>
     <!-- Show enabled lte option for lte device -->
-    <bool name="config_enabled_lte" translatable="false">false</bool>
+    <bool name="config_enabled_lte" translatable="false">true</bool>
     <!-- Show enabled tdscdma option for device -->
     <bool name="config_support_tdscdma" translatable="false">false</bool>
     <!-- Show enabled tdscdma option for device when connect roaming network -->
-- 
2.17.1

